pallets/protos/src/lib.rs

```diff
@@ -137,24 +151,33 @@ pub mod pallet {
 	use frame_system::pallet_prelude::*;
 	use pallet_detach::{DetachRequest, DetachRequests, DetachedHashes, SupportedChains};
 	use sp_clamor::CID_PREFIX;
-	use sp_runtime::{traits::Saturating, SaturatedConversion};
+	use sp_runtime::SaturatedConversion;
 
 	/// Configure the pallet by specifying the parameters and types on which it depends.
 	#[pallet::config]
 	pub trait Config:
-		frame_system::Config + pallet_detach::Config + pallet_accounts::Config
+		frame_system::Config
+		+ pallet_detach::Config
+		+ pallet_accounts::Config
+		+ pallet_assets::Config
+		+ pallet_contracts::Config
 	{
 		/// Because this pallet emits events, it depends on the runtime's definition of an event.
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
 
 		type WeightInfo: WeightInfo;
 
+		/// Const per byte multiplier
 		#[pallet::constant]
 		type StorageBytesMultiplier: Get<u64>;
```

Comment:
I have no idea what this does by the way 😅. But it's okay. I'll figure it out later

Reply:
Increases the weight of data bytes, limits storage amount x block/extrinsic

Reply:
`StorageBytesMultiplier` is basically the weight for adding a byte worth of `data` (the last parameter), yes?

Reply:
I'm documenting it like this in another PR:

/// Weight for adding a a byte worth of storage in certain extrinsics such as `upload()`.